chapter-05/final/money/convert_test.go

```diff
@@ -81,23 +82,22 @@ func TestConvert(t *testing.T) {
 				}
 			},
 		},
-		"Unknown currency": {
-			amount:          "10",
-			from:            "EUR",
-			to:              "SUR", // Soviet Union Rubles, long gone.
-			targetPrecision: 2,
-			rateRepo:        stubRate{err: fmt.Errorf("unknown currency")},
-			validate: func(t *testing.T, got string, err error) {
-				if !errors.Is(err, money.ErrUnknownChangeRate) {
-					t.Errorf("expected error %s, got %v", money.ErrUnknownChangeRate, err)
-				}
-			},
-		},
 	}
 
 	for name, tc := range tt {
 		t.Run(name, func(t *testing.T) {
-			got, err := money.Convert(context.Background(), tc.amount, tc.from, tc.to, tc.rateRepo)
+			// TODO not sure we should do the parsing here but fields from Number and Currency are not exposed
```

Comment:
TODO: I am not sure if we should do the parsing here in the tests but it was easier to write the tests case. Maybe I was too lazy? 
We have a NewCurrency as helper should we do the same for Number and just remove the parsing?